refactor/data/app/api/api_v0/endpoints/fs.py

```diff
@@ -33,12 +34,27 @@
 
 router = APIRouter()
 
-# TODO: Select and initialize backend from config
-backend = LocalStorageBackend(Path("data"))
-# minio_client = Minio(
-#     "localhost:9000", access_key="minioadmin", secret_key="minioadmin", secure=False
-# )
-# backend = MinioStorageBackend(minio_client)
+# TODO: Improve config management
+storage_backend_selector = os.environ.get("FS_STORAGE_BACKEND", "local")
+local_storage_root = os.environ.get("FS_LOCAL_STORAGE_ROOT", "data")
+bucket_name = os.environ.get("FS_STORAGE_BUCKET", "default")
+minio_endpoint = os.environ.get("MINIO_ENDPOINT", "localhost:9000")
+minio_access_key = os.environ.get("MINIO_ACCESS_KEY", "minioadmin")
+minio_secret_key = os.environ.get("MINIO_SECRET_KEY", "minioadmin")
+minio_use_tls = os.environ.get("MINIO_DISABLE_TLS", "").lower() not in ("true", "1")
```

Comment:
Is this the right way to manage config? I thought we had some kind of config manager that wasn't using environment variables.

I'm thinking of this comment from @wjcunningham7 https://github.com/AgnostiqHQ/covalent/pull/215#discussion_r834728936

Reply:
We currently have a config manager, but according to @AlejandroEsquivel 's comment below your referenced comment environment variables will be the way to go after supervisord is integrated.

Reply:
That said, perhaps the configuration logic might better belong in the modules of the respective backends.

Reply:
That said, the configuration logic might better belong in the modules of the respective backends.